### PR TITLE
Add ubuntu 18.04, 20.04 and debian 10 support

### PR DESCRIPTION
**Pull request by traylenator**
#### Pull Request (PR) description

Add ubuntu 18.04, 20.04 and debian 10 support
Drop puppet 5 support

### CODE REVIEW

**Comment by traylenator** [comment]
@olifre if you could take a look.

**Comment by traylenator** [comment]
@jblomer there is no cvmfs-config repo for debian ?

**Comment by olifre** [comment]
@traylenator That looks good from looking through the code! 
Since our infrastructure is not yet on Puppet 6 (@wiene and I are still working on the upgrade), I can not test it on real machines (yet). 

One thing I am not sure about is whether people may want to be able to configure 
`ensure  => refreshed`
for the `key` of the `apt::source`. 

I personally believe there is no security risk involved in refreshing, since `puppet-apt` only does it when the key is actually expired, using existing keyserver infrastructure (see https://github.com/puppetlabs/puppetlabs-apt/blob/8927af46c0bb430be677bc919323c133204a253d/lib/puppet/provider/apt_key/apt_key.rb#L179 ), but it rather saves a lot of hassle in real life. It has saved us in several cases for many repositories when they prolong their keys, which would be either an additional Puppet hack or manual work.

**Comment by traylenator** [comment]
@olifre should still work on puppet 5 , only removed really as no puppet5 avilable for 20.04.

**Comment by traylenator** [comment]
> ensure => refreshed on key.

Seems like a bad idea to me, if someone wants they can submit a patch.

**Comment by olifre** [comment]
> @olifre should still work on puppet 5 , only removed really as no puppet5 avilable for 20.04.

@traylenator Indeed, there does not seem to be anything in the code requiring Puppet 6 ;-). 
I tested this on a Debian 10 node just now, and it works fine — you also changed the repo URI to `https`, thanks! :+1:

File: data/common.yaml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-cvmfs::repo_incude_pkgs: ~
+cvmfs::repo_includepkgs: ~
```
Comment (bastelfreak) [inline data/common.yaml:1]: I think that bugfix should be a dedicated PR? It's not really related to adding Debian/Ubuntu support

Comment (traylenator) [reply data/common.yaml:1]: https://github.com/voxpupuli/puppet-cvmfs/pull/126

File: metadata.json [hunk 2]
```diff
@@ -41,7 +58,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 5.10.0 < 8.0.0"
+      "version_requirement": ">= 6.2.0 < 8.0.0"
```
Comment (bastelfreak) [inline metadata.json:61]: Is there a specific reason to go to 6.2.0? We usually go to 6.1.0 because that has the systemctl refresh resource embedded.

Comment (traylenator) [reply metadata.json:61]: Not at all, thought 6.2.0 was the one to go for for some reason.

**Review by bastelfreak — APPROVED** [review]
